lib/responses.js

```diff
@@ -283,17 +302,16 @@ function notFound(err, res) {
  * @param {object} res
  */
 function serverError(err, res) {
-  // error is required to be logged
-  log('error', err.message, { stack: err.stack, url: _.get(res, 'locals.url') });
-
   const message = err.message || 'Server Error', // completely hide these messages from outside
     code = 500;
 
+  log('error', err.message, { code, stack: err.stack, url: _.get(res, 'locals.url') });
```

Comment:
could use `logLevelFromStatusCode` here incase in the future you want to change behavior for 500s